### ISSUE TITLE
Cannot "ask deploy"... Saying no skill.json or root! Also git error!

### ISSUE THREAD

**Issue by Anim8edTech** [alexa/skill-sample-nodejs-adventure-game#67]
Just what it says... I cannot deploy ask.

I should back up and give some background. I could not git clone the given link because such a repo does "not exist". So, instead, I cloned the skill-sample-nodejs-adventure-game repo. I did the "ask init", and then deployed. I simply get this error, or this one: `C:\Windows\System32\skill-sample-nodejs-adventure-game> ask deploy
[Error]: Failed to deploy. Please ensure the current working directory is the root of your skill project` OR `[Error]: Can't find skill.json in current working directory`
I've used the ASK CLI before, with no issues. What's up this time? Any assistance would be amazing!

**Comment by error404notfound** [comment]
what directory are you doing the deploy from? does 
C:\Windows\System32\skill-sample-nodejs-adventure-game
actually have a skill.json in the folder?  

DId you try and clone the whole cookbook? When I first tried I found I had a copy error once i fixed that it worked.  I just  tried it with this command and was able to clone the whole thing... 

 git clone https://github.com/alexa/alexa-cookbook

**Comment by Anim8edTech** [comment]
I am deploying it from both C:\Windows\System32\skill-sample-nodejs-adventure-game and from C:\Windows\System32\skill-sample-nodejs-adventure-game\lambda\custom. I have tried `ask init` in both directories. C:\Windows\System32\skill-sample-nodejs-adventure-game does indeed have a skill.json, and I opened it, and it did have the code necessary for a skill. I did not try to clone the whole cookbook, as all I wanted was the adventure game sample. Should that bring better results? Thanks...

**Comment by escottalexander** [comment]
I have the exact same error with the exact same git issue on the original file on MacOS High Sierra.

**Comment by Anim8edTech** [comment]
Would you happen to know where to go from there? I need a project to be done by the end of April using this, and am lost on where to go from here.

**Comment by falent** [comment]
any update? I got the same issue

`[Error]: Failed to deploy. Please ensure current working directory is the root of your skill project.`

**Comment by Anim8edTech** [comment]
No, I haven't gotten any further. I do still need help with this though. Would be nice if Amazon kept up their support here.

**Comment by franklin-lobb** [comment]
I'm investigating and will post an update when I find the root cause.

**Comment by franklin-lobb** [comment]
The skill will now deploy from the command line using the 'ask deploy' command.  Can you please try again and let me know if it resolved your issue?

**Comment by falent** [comment]
I discribed my solution here: https://stackoverflow.com/questions/50040138/alexa-ask-cli-how-to-add-a-custom-https-server/50040830?noredirect=1#comment87110040_50040830 Last answer. It works for me :)

**Comment by Anim8edTech** [comment]
I'll try once I am available.

**Comment by Anim8edTech** [comment]
No help. Still stuck.

**Comment by Anim8edTech** [comment]
It still doesn't find the skill.json. I am very confused now, since it is clearly there.

**Comment by JofCla** [comment]
I have the same issues, did you figure out the problem?
Thank you!

**Comment by incisor** [comment]
the latest commit should work fine. The issue is .ask/config is missing before.

**Comment by daleyarborough** [comment]
So the issue that I ran into was not changing the directory once I had installed npm. You then have to go back to the root directory where the skill.json file is located (you can find the .ask, .config, etc. here -- if you're new to this, try typing in cd .. (do this two times) to get back to the root directory once you've installed npm). Then you will be able to deploy with the command ask deploy once you are in the root directory. Hope this helps and doesn't further confuse anyone!

**Comment by meneimoh** [comment]
> So the issue that I ran into was not changing the directory once I had installed npm. You then have to go back to the root directory where the skill.json file is located (you can find the .ask, .config, etc. here -- if you're new to this, try typing in cd .. (do this two times) to get back to the root directory once you've installed npm). Then you will be able to deploy with the command ask deploy once you are in the root directory. Hope this helps and doesn't further confuse anyone!

This worked for me. Thanks
